### Footstep parameterization for a large humanoid, similar to ASIMO or HRP-2 #

### footstep settings #########################################################
# Footstep displacement vectors (in meter / rad)
# WARNING: data type has to be float/double, integers will cause an exception
footsteps:
  x:     [0.00, 0.22, 0.00,-0.08, 0.12, 0.15, 0.08,-0.04,-0.10, 0.00, 0.15, 0.12, 0.12, 0.06]
  y:     [0.14, 0.14, 0.26, 0.12, 0.22, 0.11, 0.22, 0.22, 0.14, 0.12, 0.14, 0.12, 0.18, 0.14]
  theta: [0.00, 0.00, 0.00, 0.70, 0.30,-0.40, 0.00, 0.30, 0.00, 0.00, 0.00, 0.00, 0.00,-0.25]


### foot settings #############################################################
foot:
  size: {x: 0.16, y: 0.10, z: 0.015}
  separation: 0.14
  origin_shift: {x: 0.02, y: 0.0}
  max:
    step: {x: 0.22, y: 0.28, theta: 0.7}
    inverse:
      step: {x: -0.1, y: 0.12, theta: -0.4}


### step range ################################################################

# more accurate than the bounding box defined by the max/max inverse step
step_range:
  x: [0.22, 0.22,-0.10,-0.10]
  y: [0.28, 0.12, 0.12, 0.28]

